src/pybind/mgr/restful/common.py

```diff
@@ -34,7 +36,7 @@
 def humanify_command(command):
     out = [command['prefix']]
 
-    for arg, val in command.iteritems():
+    for arg, val in six.iteritems(command):
```

Comment:
if this is not critical, we might just use

```suggestion
    for arg, val in command.items():
```

instead.

Reply:
i am fine with either of these two approaches as long as we don't care about creating a copy of `command` in python2.